bosh-setup/azuredeploy.json

```diff
@@ -0,0 +1,341 @@
+                "Standard_D11",
+                "Standard_D12",
+                "Standard_D13",
+                "Standard_D14",
+                "Standard_DS1",
+                "Standard_DS2",
+                "Standard_DS3",
+                "Standard_DS4",
+                "Standard_DS11",
+                "Standard_DS12",
+                "Standard_DS13",
+                "Standard_DS14",
+                "Standard_G1",
+                "Standard_G2",
+                "Standard_G3",
+                "Standard_G4",
+                "Standard_G5",
+                "Basic_A0",
+                "Basic_A1",
+                "Basic_A2",
+                "Basic_A3",
+                "Basic_A4"
+            ],
+            "metadata": {
+                "description": "Size of vm"
+            }
+        },
+        "adminUsername": {
+            "type": "string",
+            "metadata": {
+                "description": "User name for the Virtual Machine."
+            }
+        },
+        "adminPassword": {
+            "type": "securestring",
+            "metadata": {
+                "description": "Password for the Virtual Machine."
+            }
+        },
+        "enableDNSOnDevbox": {
+            "type": "bool",
+            "allowedValues": [
+                true,
+                false
+            ],
+            "metadata": {
+                "description": "A default DNS will be setup in the devbox if it is true. Need a manual reboot to take effect."
+            }
+        }
+    },
+    "variables": {
+        "api-version": "2015-05-01-preview",
+
+        "vmStorageAccountContainerName": "vhds",
+        "storageAccountType": "Standard_LRS",
+        "storageid": "[resourceId('Microsoft.Storage/storageAccounts', parameters('newStorageAccountName'))]",
+
+        "virtualNetworkName": "[parameters('virtualNetworkName')]",
+        "addressPrefix": "10.0.0.0/16",
+        "vnetID": "[resourceId('Microsoft.Network/virtualNetworks', variables('virtualNetworkName'))]",
+        "subnet1Name": "[parameters('subnetNameForBosh')]",
+        "subnet1Prefix": "10.0.0.0/24",
+        "subnet1Ref": "[concat(variables('vnetID'),'/subnets/', variables('subnet1Name'))]",
+        "subnet2Name": "[parameters('subnetNameForCloudFoundry')]",
+        "subnet2Prefix": "10.0.16.0/24",
+        "nicName": "[parameters('vmName')]",
+        "devboxPrivateIPAddress": "10.0.0.100",
+        "devboxPublicIPAddressID": "[resourceId('Microsoft.Network/publicIPAddresses', concat(parameters('vmName'),'-devbox'))]",
+
+        "imagePublisher": "Canonical",
+        "imageOffer": "UbuntuServer",
+        "ubuntuOSVersion": "14.04.2-LTS",
+
+        "repository": "Azure",
+        "branch": "master",
+        "githubPath": "[concat('https://raw.githubusercontent.com/', variables('repository'), '/azure-quickstart-templates/', variables('branch'), '/bosh-setup/')]",
+        "cloudfoundryBlob": "http://cloudfoundry.blob.core.windows.net/misc/"
```

Comment:
Because bosh.yml maybe changes frequently, especially in Beta version. So we keep them in our Azure storage. The storage is public. If users want to view the file, they can download them by the url.